lib/charms/tls_certificates_interface/v3/tls_certificates.py

```diff
@@ -450,6 +450,10 @@ class ProviderCertificate:
     chain: List[str]
     revoked: bool
 
+    def chain_as_pem(self) -> str:
+        """Return full certificate chain as a PEM string."""
+        return "\n\n".join(reversed(self.chain))
```

Comment:
Should we reconsider the chain being a list of string in general? Is it being a pem string more valuable for most cases?

Reply:
This could work as an approach to have PEM by default, and optionally have a method or property to get the chain as a list. The main concern would be breaking the public API and having to create a new major version again.